doc/dev/radosgw/s3_objects_dedup.rst

```diff
@@ -0,0 +1,60 @@
+======================
+Full RGW Object Dedup:
+======================
+Add a radosgw-admin command to collect and report deduplication stats
+
+.. note:: This utility doesn’t perform dedup and doesn’t make any
+          change to the existing system and will only collect
+          statistics and report them.
+
+----
+
+***************
+Admin commands:
+***************
+- ``radosgw-admin dedup stats``:
+   Collects & displays last dedup statistics
+- ``radosgw-admin dedup pause``:
+   Pauses active dedup session (dedup resources are not released)
+- ``radosgw-admin dedup resume``:
+   Resumes a paused dedup session
+- ``radosgw-admin dedup abort``:
+   Aborts active dedup session and release all resources used by it
+- ``radosgw-admin dedup restart estimate``
+    Starts a new dedup estimate session (aborting first existing session if exists)
+
+----
+
+*************
```

Comment:
i'm not sure the "Memory Usage" belong under the user doc.
the end user will care more about 
* the output of the command, as this is what they will look to understand is their system would benefit from dedup.
limitation on dedup
* minimal size to dedup
* restrictions on storage classes
* o we dedup across buckets? users? tenats?
* do we dedup encrypted ot compressed objects?
* versioned buckets?
* multipart objects?
* md5 vs. sha256 as dedup criteria